src/components/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ name, color, wide }) => (
+  <div className="button" style={{ backgroundColor: color, flex: wide }}>{name}</div>
+);
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  color: PropTypes.string,
+  wide: PropTypes.number,
+};
```

Comment:
`wide` is a boolean instead 

==> https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png